stylesheets/modules/_navigation.scss

```diff
@@ -1,6 +1,6 @@
 .site-navigation {
-  line-height: 2.5;
-  margin: 0 auto;
+  line-height: 3;
+  margin: 0 auto 0.125em auto;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 auto 0.125em` instead of `0 auto 0.125em auto`